blacklight-cornell/features/catalog_search/facets.feature

```diff
@@ -130,3 +130,7 @@ Feature: Facets
 		| lc_callnum_facet | Call Number | x |
 		| collection | Collection | x |
 		# wip | acquired_dt_query | Date Acquired | yes |
+
+Scenario: Filtering searches by facets that are not available in the public United
```

Comment:
Nooo, I missed this typo (this should have said `ui` but somehow got autocorrected?!). Fixing now...